src/libraries/System.DirectoryServices/Directory.Build.props

```diff
@@ -7,5 +7,6 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
     <IsWindowsSpecific>true</IsWindowsSpecific>
+    <IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>
```

Comment:
This is going to include the attribute even for tests. Would be better to only mark specific projects with this?

I see `ref/System.DirectoryServices.csproj` is marked with this already (not sure whether it is required or oversight).

Reply:
Oh, of course -- thanks.

Reply:
And to answer: it was oversight.  I started by adding it to `Directory.Build.props` but then pivoted to applying it in the projects and forgot to remove it.